### DISCUSSION TITLE
Is `smoothscroll` supported?

### DISCUSSION THREAD

**Discussion by qadzek** [vscode-neovim/vscode-neovim#1970] [Q&A] [answered]
Nvim v0.10 supports scrolling by screen lines if a line wraps, rather than by line number.
Is this supposed to be working in VS Code as well?
I tried `set smoothscroll`, but `<C-e>` and `<C-y>` seem to work as usual.

**Comment by theol0403** [comment, marked as answer]
No, for many reasons. This functionality can theoretically be implemented but requires a lot of work (scroll sync and fold support). But synchronizing with nvim settings is the least of our worries.

**Comment by xiyaowong** [comment]
If you really need this option, you should make a feature request to vscode.
